Hestia/app/src/main/java/hestia/UI/dialogs/AddDeviceDialog.java

```diff
@@ -50,7 +50,7 @@ public void setFragmentManager(FragmentManager fragmentManager) {
 
     @Override
     String buildTitle() {
-        return "new Add device";
+        return getContext().getString(R.string.addDeviceTitle);
```

Comment:
This is my mistake. I changed the name to " new Add device" to see whether or not the dialog actually changed. Forgot to change it back.